redis.conf

```diff
@@ -891,14 +891,11 @@ acllog-max-len 128
 # Both LRU, LFU and volatile-ttl are implemented using approximated
 # randomized algorithms.
 #
-# Note: with any of the above policies, Redis will return an error on write
-#       operations, when there are no suitable keys for eviction.
-#
-#       At the date of writing these commands are: set setnx setex append
-#       incr decr rpush lpush rpushx lpushx linsert lset rpoplpush sadd
-#       sinter sinterstore sunion sunionstore sdiff sdiffstore zadd zincrby
-#       zunionstore zinterstore hset hsetnx hmset hincrby incrby decrby
-#       getset mset msetnx exec sort
+# Note: with any of the above policies, when there are no suitable keys for
+#       eviction, Redis will return an error on write operations that require
+#       more memory, like creating a new key (like for example the "set"
+#       command), or increasing the size of an existing data structure (like
+#       for example the "lpush" command).
```

Comment:
I'll try to improve let's see if it works better.
p.s. `sdiff` and `sinter` where a bug. they don't have the `use-memory` flag.
```suggestion
# Note: with any of the above policies, when there are no suitable keys for
# eviction, Redis will return an error on write operations that require
# more memory. These are usually commands that create new keys, add data or
# modify existing keys. A few examples are: SET, INCR, HSET, LPUSH, SUNIONSTORE,
# SORT (due to the STORE argument), and EXEC (if the transaction includes any
# command that requires memory).
```

Reply:
Great, just changed to your text!